packages/sveltekit/src/vite/sentrySvelteKitPlugin.ts

```diff
@@ -0,0 +1,54 @@
+import type { Plugin, UserConfig } from 'vite';
+
+import { injectSentryInitPlugin } from './injectInitPlugin';
+import { hasSentryInitFiles } from './utils';
+
+/**
+ * Vite Plugin for the Sentry SvelteKit SDK, taking care of:
+ *
+ * - Creating Sentry releases and uploading source maps to Sentry
+ * - Injecting Sentry.init calls if you use dedicated `sentry.(client|server).config.ts` files
```

Comment:
should we think about removing this injection behaviour all together then?

Reply:
Yeah I was sort of debating if we should keep it as an alternative but I think the drawback is that we have to support both ways. Also the init location of hook vs. injected init calls differs slightly in the final bundles. Since IMO hooks are the better choice anyway, I'd say we remove it. 

But I suggest we do it in a follow up PR, to make reverting easier if we have to, wdyt?

Reply:
let's do it in a follow up PR!